array-foreach/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "noImplicitAny": true,
+        "strictNullChecks": false,
```

Comment:
apologies, copied the array-find-index directory above and started from their, I assumed the tsconfig would be correct, guessing it's been changed over time. Updated as per the one in the readme